ddtrace/bootstrap/sitecustomize.py

```diff
@@ -0,0 +1,16 @@
+from __future__ import print_function
+
+"""
+Bootstrapping code that is run when using the `ddtrace-run` Python entrypoint
+Add all monkey-patching that needs to run by default here
+"""
+
+try:
+    from ddtrace import patch_all; patch_all(django=True, flask=True, pylons=True) # noqa
```

Comment:
let's add an env var which can disalbe this.

`DATADOG_TRACE_ENABLED=... ddtrace run my_script.py`